api/src/main/java/uk/gov/dvsa/mot/mottest/api/MotTestRfrLocation.java

```diff
@@ -11,11 +11,11 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "", propOrder = {"lateral", "longitudinal", "vertical"})
 public class MotTestRfrLocation {
-    @XmlElement(name = "lateral", required = false)
+    @XmlElement(name = "lateral")
```

Comment:
IntelliJ inspector picked it up as redundant because the default value is `false`.